### PR TITLE
Cudos 1600 additional nft methods

### PR DESCRIPTION
**Pull request by Infra1515**
Methods for aura payment service

### CODE REVIEW

File: x/nft/client/rest/query.go [hunk 1]
```diff
@@ -57,6 +58,57 @@ func registerQueryRoutes(cliCtx client.Context, r *mux.Router) {
...
+
+		var req queryCollectionsByDenomIdsRequest
+		if !rest.ReadRESTReq(w, r, cliCtx.LegacyAmino, &req) {
+			rest.WriteErrorResponse(w, http.StatusBadRequest, "failed to parse request")
+			return
+		}
+
+		for _, id := range req.DenomIds {
+			if err := types.ValidateDenomID(id); err != nil {
+				rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
```
Comment (avalkov) [inline x/nft/client/rest/query.go:84]: Missing return, I see a few other similar cases in the already committed REST methods.

File: x/nft/client/rest/query.go [hunk 2]
```diff
@@ -57,6 +58,57 @@ func registerQueryRoutes(cliCtx client.Context, r *mux.Router) {
...
+func queryCollectionsByDenomIds(cliCtx client.Context) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+
+		var req queryCollectionsByDenomIdsRequest
+		if !rest.ReadRESTReq(w, r, cliCtx.LegacyAmino, &req) {
+			rest.WriteErrorResponse(w, http.StatusBadRequest, "failed to parse request")
+			return
+		}
+
+		for _, id := range req.DenomIds {
```
Comment (avalkov) [inline x/nft/client/rest/query.go:82]: Maybe we wanna check if req.DenomIds is not empty

File: x/nft/client/rest/query.go (hunk 1 above)
Comment (Infra1515) [reply x/nft/client/rest/query.go:84]: fixed

File: x/nft/client/rest/query.go (hunk 2 above)
Comment (Infra1515) [reply x/nft/client/rest/query.go:82]: fixed

File: x/nft/client/rest/query.go (hunk 1 above)
Comment (Infra1515) [reply x/nft/client/rest/query.go:84]: Also, please if you can pinpoint exactly where the returns are missing

**Pull request merged**